### PR TITLE
Improve contributing.md

### PR DESCRIPTION
**Pull request by Faisal-Alsrheed**
I've made small updates to `CONTRIBUTING.md` to improve clarity and ease for new contributors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+Keras 3 is a high-velocity open-source project. We welcome contributions!
+
+Contributions can be made in a variety of ways, including coding, enriching documentation, refining docstrings, and offering examples of code.
+
+
+## The Current Opportunities to Contribute
```
Comment (fchollet) [inline CONTRIBUTING.md:6]: -> "Current items open for contributions"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+Check this [link](https://github.com/keras-team/keras/issues/18442) to find the current opportunities to contribute.
```
Comment (fchollet) [inline CONTRIBUTING.md:7]: At [this link](https://github.com/keras-team/keras/issues/18442), you'll find a list of items where you help is needed!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+Keras 3 is a high-velocity open-source project. We welcome contributions!
+
+Contributions can be made in a variety of ways, including coding, enriching documentation, refining docstrings, and offering examples of code.
```
Comment (fchollet) [inline CONTRIBUTING.md:3]: offering examples of code -> providing code examples.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Comment by Faisal-Alsrheed** [comment]
Done, @fchollet :)
Thank you for the feedback.

**Review by fchollet — APPROVED** [review]

**Pull request merged**